sql_export_mail/models/sql_export.py

```diff
@@ -138,13 +146,38 @@ def check_mail_user(self):
                 if not user.email:
                     raise UserError(_("The user does not have any e-mail address."))
 
+    @api.constrains("mail_partner_ids")
```

Comment:
I guess the constrains should be raised, when query is changed. Or did I missed something ?